### ISSUE TITLE
Exchange Token Endpoint is Broken

### ISSUE THREAD

**Issue by MakerScott** [travoltron/plaid#9]
The exchangeToken endpoint should point to: "item/public_token/exchange" not "exchange_token". I assume the API changed at some point.

**Comment by travoltron** [comment]
Which branch did you install from? 2.0 is under some heavy modification all over the place, so it may be a little wonky. 

In the meantime, I'll take a look at that endpoint and see if I can patch that for you.

**Comment by MakerScott** [comment]
Master branch. It looks like all the endpoints need to change. I would take a look at that. It's my first time working with Plaid, so I'm not sure if they pushed breaking changes. I also didn't need to call the tartan URI in config, and I changed that to sandbox. The sample JS they provide on their site to launch Link uses the sandbox environment. The token can only be exchanged using the sandbox URI on the server side, and using my own client_id and secret vars.

**Comment by travoltron** [comment]
Yeah, looks like I'm operating on a legacy branch. This all is a wholesale change, with each product now being a distinct `ITEM` , and having its own access_token. Hrmmm. It might be a minute while I redo this. Thanks for the heads up.

**Comment by travoltron** [comment]
FWIW, I didn't get any notice from Plaid and I use their API in two different projects, and my existing connection works without trouble. But these all represent breaking changes. 

However, they've been nothing short of amazing to work with thus far.

**Comment by MakerScott** [comment]
Interesting. I definitely got it working by adjusting these settings. It's possible they may have a legacy API that still works for older customers.

**Comment by travoltron** [comment]
No doubt, otherwise I'd have many other fires burning...

Oh well, time to revamp.
